geometry_msgs/Point upleftPoint
geometry_msgs/Point downrightPoint
